### PR TITLE
[DCOS-41744] add hello world metrics smoke test

### PR DESCRIPTION
**Pull request by elezar**
This adds a metrics smoke test to Hello-world and should fail with the current master build.

TODO:
- [ ] Ensure that the list of metrics are sane.
- [ ] Switch to getting the scheduler metrics here including those adding in #2538

### CODE REVIEW

**Comment by porridge** [comment]
For posterity, might be worth mentioning this is related to https://jira.mesosphere.com/browse/DCOS-41416

File: frameworks/helloworld/tests/test_sanity.py [hunk 1]
```diff
@@ -251,15 +289,33 @@ def test_plan_cli():
...
     _check_json_output(foldered_name, "plan show --json {}".format(plan_name))
     _check_json_output(foldered_name, "plan show {} --json".format(plan_name))
 
     # trigger a restart so that the plan is in a non-complete state.
     # the 'interrupt' command will fail if the plan is already complete:
-    assert 0 == sdk_cmd.svc_cli(config.PACKAGE_NAME, foldered_name, "plan force-restart {}".format(plan_name))[0]
-    assert 0 == sdk_cmd.svc_cli(config.PACKAGE_NAME, foldered_name, "plan interrupt {} {}".format(plan_name, phase_name))[0]
-    assert 0 == sdk_cmd.svc_cli(config.PACKAGE_NAME, foldered_name, "plan continue {} {}".format(plan_name, phase_name))[0]
+    assert (
+        0
```
Comment (kvish) [inline frameworks/helloworld/tests/test_sanity.py]: spacing here

File: frameworks/helloworld/tests/test_sanity.py [hunk 2]
```diff
@@ -234,12 +269,15 @@ def test_config_cli():
...
     assert len(configs) >= 1  # refrain from breaking this test if earlier tests did a config update
 
-    assert 0 == sdk_cmd.svc_cli(
-        config.PACKAGE_NAME,
-        foldered_name,
-        "debug config show {}".format(configs[0]),
-        print_output=False,  # noisy output
-    )[0]
+    assert (
+        0
```
[the remark this answers is not in the document]
Comment (elezar) [reply frameworks/helloworld/tests/test_sanity.py]: Yes. This was formatted with `black`.

File: frameworks/helloworld/tests/test_sanity.py (hunk 1 above)
Comment (elezar) [reply frameworks/helloworld/tests/test_sanity.py]: Autoformatted with `black`.

**Comment by elezar** [comment]
@kvish I have reworked the spots where the formatting was off.

**Pull request closed without merging**